Mathlib/LinearAlgebra/Matrix/ToLin.lean

```diff
@@ -955,3 +955,4 @@ lemma end_apply_apply (ij : ι × ι) (k : ι) : (b.end ij) (b k) = if ij.2 = k
   linearMap_apply_apply b b ij k
 
 end Basis
+smulTowersmulTowersmulTowersmulTower
```

Comment:
I think the F2 rename tool is stupidly broken.

Reply:
Could you open an issue for this?